styles/raindrop/catppuccin.user.css

```diff
@@ -0,0 +1,176 @@
+/* ==UserStyle==
+@name Raindrop Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/raindrop
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/raindrop
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/raindrop/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Araindrop
+@description Soothing pastel theme for Raindrop
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('app.raindrop.io') {
+  :root[data-theme="night"] {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+  :root[data-theme="day"] {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+  :root[data-theme="sunset"] {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  // theme selector previews
+  [class^="selector-"] > [class^="theme-"][data-index="0"] {
+    #catppuccin(@lightFlavor, @accentColor) !important;
+  }
+  [class^="selector-"] > [class^="theme-"][data-index="1"] {
+    #catppuccin(@darkFlavor, @accentColor) !important;
+  }
+  [class^="selector-"] > [class^="theme-"][data-index="2"] {
+    [class^="sidebar-"] {
+      #catppuccin(@darkFlavor, @accentColor) !important;
+    }
+    [class^="main-"] {
+      #catppuccin(@lightFlavor, @accentColor) !important;
+    }
+  }
```

Comment:
Can we use CSS nesting to nest this a little more?

Reply:
is this better?
```css
  // theme selector previews
  .night_theme{
    #catppuccin(@darkFlavor, @accentColor) !important;
  }
  .day_theme{
    #catppuccin(@lightFlavor, @accentColor) !important;
  }
  [class^="theme-"] {
    &[data-index="0"] {
      .day_theme()
    }
    &[data-index="1"] {
      .night_theme()
    }
    &[data-index="2"] {
      [class^="sidebar-"] {
        .night_theme()
      }
      [class^="main-"] {
        .day_theme()
      }
    }
  }
```